docs/frameworks/angular.md

```diff
@@ -0,0 +1,160 @@
+---
+description: Learn how to deploy a Angular application with Genezio.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Angular
+
+<head>
+    <title>Angular | Genezio Documentation</title>
+</head>
+
+[Angular](https://angular.io/) is a popular JavaScript framework for building single-page applications. It is known for its robust features and scalability, making it a great choice for developers who want to build modern web applications.
+
+:::tip
+Get started in no time with the [Angular template](https://app.genez.io/start/deploy?repository=https://github.com/Genez-io/angular-getting-started).
+:::
+
+# Deployment
+
+Learn how to deploy an existing Angular app using Genezio, a serverless deployment platform that simplifies app management and reduces costs.
+
+## Prerequisites
+
+### 1. Install genezio
+
+Use your preferred package manager to install Genezio:
+
+<Tabs>
+  <TabItem className="tab-item" value="npm" label="npm">
+<div id="step1-install-npm">
+  ```
+  npm install genezio -g
+  ```
```

Comment:
I'm wondering if we should teach people how to use Build Machine instead 🤔.

Reply:
I mentioned this at the beginning:

`:::tip
Get started in no time with the [Angular template](https://app.genez.io/start/deploy?repository=https://github.com/Genez-io/angular-getting-started).
:::`

I think that's enough

Reply:
ok